benchmark,doc: add CPU scaling governor to perf (#54723)
nodejs/node: doc/contributing/writing-and-running-benchmarks.md

## Running benchmarks

### Setting CPU Frequency scaling governor to "performance"

> It is recommended to set the CPU frequency to 'performance' before running
> benchmarks. This ensures that each benchmark run can achieve peak performance
> according to the hardware. Therefore, run:

Reviewer: ```suggestion
It is recommended to set the CPU frequency to `performance` before running
benchmarks. This increases the likelihood of each benchmark achieving peak performance
according to the hardware. Therefore, run:
```
